### PR TITLE
fix: Limit added to api routes

### PR DESCRIPTION
**Pull request by Ambrdctr**

### CODE REVIEW

File: MailCore/Cache/MailboxManager/MailboxManager+Message.swift [hunk 1]
```diff
@@ -127,34 +127,35 @@ public extension MailboxManager {
...
             }
         } catch {
             await updateLocally(.star, value: !starred, messages: messages)
         }
     }
 
-    private func star(messages: [Message]) async throws -> MessageActionResult {
-        let response = try await apiFetcher.star(mailbox: mailbox, messages: messages)
+    private func star(messages: [Message]) async throws {
+        _ = try await apiFetcher.star(mailbox: mailbox, messages: messages)
```
Comment (adrien-coye) [inline MailCore/Cache/MailboxManager/MailboxManager+Message.swift:140]: maybe mark `star` and `unstar` as discardable results ?

**Review by adrien-coye — COMMENTED** [review]
Looking good. I only have some small remarks.

File: MailCore/API/MailApiFetcher/MailApiFetcher.swift [hunk 2]
```diff
@@ -86,4 +88,19 @@ public final class MailApiFetcher: ApiFetcher, MailApiFetchable {
...
     }
+
+    /// Create batches of the given values to perform the given request
+    /// - Parameters:
+    ///   - values: Data to batch
+    ///   - chunkSize: Chunk size
+    ///   - perform: Request to perform
+    /// - Returns: Array of the perform return type
+    func batchOver<Input, Output>(values: [Input], chunkSize: Int,
+                                  perform: @escaping (([Input]) async throws -> Output?)) async rethrows -> [Output] {
```
Comment (PhilippeWeidmann) [inline MailCore/API/MailApiFetcher/MailApiFetcher.swift:99]: Why do we allow nil output if we end up using compactMap ?

**Review by adrien-coye — APPROVED** [review]
All previous discussions are closed. Looks good to me 👍

**Review by PhilippeWeidmann — APPROVED** [review]

**Pull request merged**